website/docs/api/scorer.md

```diff
@@ -46,7 +46,9 @@ attribute being scored:
 - `tag_acc`, `pos_acc`, `morph_acc`, `morph_per_feat`, `lemma_acc`
 - `dep_uas`, `dep_las`, `dep_las_per_type`
 - `ents_p`, `ents_r` `ents_f`, `ents_per_type`
-- `textcat_macro_auc`, `textcat_macro_f`
+- `cats_score`, `cats_score_desc`, `cats_micro_p`, `cats_micro_r`,
```

Comment:
it's a little awkward to have `cats_score_desc` in this list?

Reply:
I went back and forth about this, but just included all the fields in the end. I don't have a strong opinion either way, but it seems helpful to know it's there?

Reply:
I'd be inclined to maybe mention it separately, because the text says "attributes being scored" and this one is a string. Perhaps this particular one requires a bit more explanation?

We could keep it in the list but include a note?